packages/rich-text-from-markdown/src/__test__/real-world.md

```diff
@@ -88,7 +89,7 @@ Inline `code`
 | Test 1                                       | Germany |
 | Test 2                                       | USA     |
 | > Test 3                                     | USA     |
-| * Test 4                                    | Germany |
+| \* Test 4                                    | Germany |
```

Comment:
what does this need to be escaped now?

Reply:
It doesn't, for some reason even when I save without formatting it still commits it with the escaped character :( tests are still passing so it doesn't seem to make a difference though

Reply:
That's prettier's doing. Even if you save without formatting we have lint-staged.

Reply:
you could disable prettier around this part instead of escaping https://prettier.io/docs/en/ignore.html#markdown

Reply:
We already have prettier disabled in that way for the whole file, but it doesn't seem to be working anymore